Source/Machine.Fakes.Adapters.Specs/InlineConstraintSpecs.tt

```diff
@@ -6,7 +6,7 @@
 <#@ output extension=".generated.cs" #>
 <# var allEngines = new[] { "RhinoFakeEngine", "NSubstituteEngine", "MoqFakeEngine", "FakeItEasyEngine" };  #>
 <# var allNamespaces = new[] { "Rhinomocks", "NSubstitute", "Moq", "FakeItEasy" };  #>
-<# var supportsNetStandard = new[] { false, false, true, false };  #>
+<# var supportsNetStandard = new[] { false, true, true, true };  #>
```

Comment:
It looked to me like the generated test file was saved, adding NSubstitute and FakeItEasy to those tested under .NET Standard, but this file wasn't updated. Changing these values here mean we _keep_ testing those two frameworks under .NET Standard.